components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.geo-dashboard/public/js/geo_fencing.js

```diff
@@ -431,3 +431,76 @@ function viewFence(geoFenceElement,id) {
     }
     closeAll();
 }
+
+function viewFenceByData(geoJson, queryName, areaName, stationeryTime, id) {
+    var matchResults = /(?:"geoFenceGeoJSON"):"{(.*)}"/g.exec(geoJson);
+    if (matchResults && matchResults.length > 1) {
+        geoJson = "{" + matchResults[1] + "}";
+    }
+    geoJson = JSON.parse(geoJson.replace(/'/g, '"'));
+    var geometryShape;
+
+    if(geoJson.type=="Point"){
```

Comment:
Apply the code formatting to the code.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/950